README.md

```diff
@@ -1,7 +1,128 @@
+
+    In the process of developing these building blocks, we will by necessity implement end-to-end 
+    workflows, but they're intended purely for demonstration and grounding purposes, they're not 
+    our main deliverable.
+
+
+- **KerasCV is not a repository of low-level image-processing ops, like tf.vision.** 
+
+    KerasCV is fundamentally an extension of the Keras API: it hosts Keras objects, like layers, 
+    metrics, or callbacks. Low-level C++ ops should go elsewhere.
+
+
+- **KerasCV is not a research library.**
+
+    Researchers may use it, but we do not consider researchers to be our target audience. Our target 
+    audience is applied CV engineers with experimentation and production needs. KerasCV should make 
+    it possible to quickly reimplement industry-strength versions of the latest generation of 
+    architectures produced by researchers, but we don't expect the research effort itself to be built
+    on top of KerasCV. This enables us to focus on usability and API standardization, and produce 
+    objects that have a longer lifespan than the average research project.
+
+## Areas of focus
+At the start of the project, we would like to take a task-oriented approach to outline our project, 
+and the first 2 tasks we would like to focus are:
+
+1. **Image classification** (Given an image and a list of labels, return the label and confidence for 
+   the image).
+2. **Object detection** (Given an image and a list of labels, return a list of bounding box, 
+   corresponding labels and confidence for the objects in the image).
+
+For each of the tasks, we are targeting to provide user model architectures, layers, preprocessing
+functions, losses, metrics and other necessary components to achieve state of the art accuracy.
+
+There will be common shareable components between those 2 tasks, like model building blocks, etc. We
+will have them in the repository as public API as well.
+
+### Common components
+- Model building block (served as backbone for the application)
+  - Resnet 
+  - Resnet RS
+  - ResNeXT
+  - Efficient
+- Preprocessing layers
+  - CutMix
+  - MixUp
+  - CutOut
+  - RandAugment
+  - RandomErasing
+
+### Image Classification
+- Dataset to evaluate
+  - [Imagenet](https://www.tensorflow.org/datasets/catalog/imagenet2012)
+  - [Imagenet v2](https://www.tensorflow.org/datasets/catalog/imagenet_v2) (for testing)
+- Target Metric 
+  - Top1 Accuracy > 80% (Note that some existing SotA models might not achieve this, like 
+    original ResNet, but this serve as a guideline for newly added models)
+
+### Object Detection
+- Dataset to evaluate
+  - [Coco 2017](https://www.tensorflow.org/datasets/catalog/coco#coco2017)
+  - [Open image v4](https://www.tensorflow.org/datasets/catalog/open_images_v4)
+  - [PASCAL voc](https://www.tensorflow.org/datasets/catalog/voc)
+- Target Metric
+  - COCO metric average precision (AP)
+- Detection model
```

Comment:
If you want we can leave the components in that we intend to add - but I'd personally opt to keep the README up to date iwth current info, and have a ROADMAP.md file with future info.

Reply:
sure. Remove the roadmap related items for now.